test/simulator-e2e-specs.js

```diff
@@ -360,6 +361,74 @@ function runTests (deviceType) {
       await getTouchEnrollKeys().should.eventually.deep.equal(originalValues);
     });
   });
+
+  describe(`multiple instances of ${deviceType.version} simulator on Xcode9+`, function () {
+    this.timeout(LONG_TIMEOUT);
+    const simulatorsMapping = new Map();
+    const DEVICES_COUNT = 2;
+    const verifyStates = async (sim, shouldServerRun, shouldClientRun) => {
```

Comment:
This will eventually have to be pulled out to a more accessible place, since the headless test also uses it.